pkg/sds/server/grpc.go

```diff
@@ -44,7 +44,7 @@ func (s *grpcServer) Start(stop <-chan struct{}) error {
 	}
 	grpcServer := grpc.NewServer(grpcOptions...)
 
-	lis, err := net.Listen("tcp", fmt.Sprintf(":%d", s.config.GrpcPort))
+	lis, err := net.Listen("tcp", fmt.Sprintf("127.0.0.1:%d", s.config.GrpcPort))
```

Comment:
Please revert this back so the server can be accessed outside of the control plane machine.